server/routes/activities/create-an-activity/handlers/removePay.ts

```diff
@@ -7,7 +7,7 @@ import { ActivityUpdateRequest } from '../../../../@types/activitiesAPI/types'
 
 export class ConfirmRemoveOptions {
   @Expose()
-  @IsIn(['yes', 'no'], { message: 'Select either yes or no' })
+  @IsIn(['yes', 'no'], { message: 'Confirm if you want to delete the pay rate or not' })
```

Comment:
I can see there's no tests for this validation currently, but would you mind adding validation tests for this form?

Just needs to confirm it's rejecting empty / invalid values and accepting valid values.

Reply:
Of course, good spot. I will update 👍